turbo.json

```diff
@@ -38,7 +38,6 @@
         },
         "publish-packages": {
             "cache": false,
-            "dependsOn": ["build", "^publish-packages"],
```

Comment:
Ooh, no. Publish really does have to depend on the upstream publishes, otherwise the wrong version numbers will get written into the package.jsons.

Reply:
In my testing it seemed that not having a `dependsOn` means that it runs all the upstream `publish-package` tasks, is that not the same as setting `"dependsOn": ["^publish-packages"]`?

Reply:
Oh, really? Oh in that case, ship it.

Reply:
Oh, I see. It will.

Reply:
I think `pnpm turbo publish-packages` won't, but `pnpm publish-packages` will